neuralprophet/plot_model_parameters_plotly.py

```diff
@@ -0,0 +1,669 @@
+            1 shifts by 1 day to Jan 2, and so on.
+        quick (bool): use quick low-evel call of model. might break in future.
+        multiplicative (bool): set y axis as percentage
+    Returns:
+        A dictionary with Plotly traces, xaxis and yaxis
+    """
+    traces = []
+    color = "#0072B2"
+    line_width = 1
+    zeroline_color = "#AAA"
+
+    # Compute yearly seasonality for a Jan 1 - Dec 31 sequence of dates.
+    days = pd.date_range(start="2017-01-01", periods=365) + pd.Timedelta(days=yearly_start)
+    df_y = pd.DataFrame({"ds": days})
+    if quick:
+        predicted = predict_season_from_dates(m, dates=df_y["ds"], name=comp_name)
+    else:
+        predicted = m.predict_seasonal_components(df_y)[comp_name]
+
+    traces.append(
+        go.Scatter(
+            name=comp_name,
+            x=range(
+                df_y["ds"].dt.to_pydatetime(),
+                y=predicted,
+                mode="lines",
+                line=dict(color=color, width=line_width),
+                fill="none",
+            ),
+        )
+    )
+
+    xaxis = go.layout.XAxis(title="Day of year")
+    yaxis = go.layout.YAxis(
+        rangemode="normal",
+        title=go.layout.yaxis.Title(text=f"Seasonality: {comp_name}"),
+        zerolinecolor=zeroline_color,
+    )
+
+    if multiplicative:
+        yaxis.update(tickformat="%", hoverformat=".2%")
+
+    return {"traces": traces, "xaxis": xaxis, "yaxis": yaxis}
+
+
+def plot_weekly(m, comp_name="weekly", weekly_start=0, quick=True, multiplicative=False):
+    """Plot the weekly component of the forecast.
+
+    Args:
+        m (NeuralProphet): fitted model.
+        comp_name (str): Name of seasonality component if previously changed from default 'weekly'.
+
+        weekly_start (int): specifying the start day of the weekly seasonality plot.
+            0 (default) starts the week on Sunday.
+            1 shifts by 1 day to Monday, and so on.
+        quick (bool): use quick low-evel call of model. might break in future.
+        multiplicative (bool): set y axis as percentage
+
+    Returns:
+        A dictionary with Plotly traces, xaxis and yaxis
+    """
+    traces = []
+    color = "#0072B2"
+    line_width = 1
+    zeroline_color = "#AAA"
+
+    # Compute weekly seasonality for a Sun-Sat sequence of dates.
+    days_i = pd.date_range(start="2017-01-01", periods=7 * 24, freq="H") + pd.Timedelta(days=weekly_start)
+    df_w = pd.DataFrame({"ds": days_i})
+    if quick:
+        predicted = predict_season_from_dates(m, dates=df_w["ds"], name=comp_name)
+    else:
+        predicted = m.predict_seasonal_components(df_w)[comp_name]
+    days = pd.date_range(start="2017-01-01", periods=7) + pd.Timedelta(days=weekly_start)
+    days = days.day_name()
+
+    traces.append(
+        go.Scatter(
+            name=comp_name,
+            x=range(len(days_i), y=predicted, mode="lines", line=dict(color=color, width=line_width), fill="none"),
+        )
```

Comment:
```suggestion
           x=list(range(len(days_i))), y=predicted, mode="lines", line=dict(color=color, width=line_width), fill="none"),
```
x cannot be the range type, should be a list